pkgs/racket-doc/scribblings/reference/hashes.scrbl

```diff
@@ -212,6 +212,13 @@ mappings can hide earlier mappings.
 
 See also @racket[make-custom-hash].
 
+@examples[
```

Comment:
The commit def72dc73b03887048e251729075dc168b486df5 is already in #4916. It looks like you accidentally include it in this PR?

Reply:
yes, that other PR was accidentally done on the master branch and seems to have carried over. Will remove this change from this PR